e2e/rbd.go

```diff
@@ -213,16 +213,34 @@ var _ = Describe("RBD", func() {
 
 			By("create a PVC and Bind it to an app", func() {
 				validatePVCAndAppBinding(pvcPath, appPath, f)
+				// validate created backend rbd images
```

Comment:
Consider making this its own function? It is repeated really often now.

Reply:
will do it in next PR as am working on refactoring the E2E tests